src/lib/markbind/src/preprocessors/componentPreprocessor.js

```diff
@@ -307,6 +302,9 @@ function _preprocessInclude(node, context, config, parser) {
   element.children = cheerio.parseHTML(childrenHtml, true);
 
   if (element.children && element.children.length > 0) {
+    childContext.source = isIncludeSrcMd ? 'md' : 'html';
+    childContext.callStack.push(context.cwf);
```

Comment:
moved this down here to where its actually needed, so the variable processing code above is cleaner